_query-dsl/query-dsl/compound/function-score.md

```diff
@@ -0,0 +1,827 @@
+            }
+          }
+        }
+      ]
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The first two blogs in the results have a score of 1 because one is at the origin (20) and the other is at a distance 16 which is within the offset (the range at which documents receive a full score is calculated as 20 $$\pm$$ 5 and is [15, 25]). The third blog is at a distance of `scale` + `offset` from the `origin` (20 &minus; (5 + 10) = 15), so it is given the default `decay` score (0.5):
+
+<details open markdown="block">
+  <summary>
+    Response
+  </summary>
+  {: .text-delta}
+
+```json
+{
+  "took": 3,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 4,
+      "relation": "eq"
+    },
+    "max_score": 1,
+    "hits": [
+      {
+        "_index": "blogs",
+        "_id": "1",
+        "_score": 1,
+        "_source": {
+          "name": "Semantic search in OpenSearch",
+          "views": 1200,
+          "likes": 150,
+          "comments": 16,
+          "date_posted": "2022-04-17"
+        }
+      },
+      {
+        "_index": "blogs",
+        "_id": "2",
+        "_score": 1,
+        "_source": {
+          "name": "Get started with OpenSearch 2.7",
+          "views": 1400,
+          "likes": 100,
+          "comments": 20,
+          "date_posted": "2022-05-02"
+        }
+      },
+      {
+        "_index": "blogs",
+        "_id": "3",
+        "_score": 0.5,
+        "_source": {
+          "name": "Distributed tracing with Data Prepper",
+          "views": 800,
+          "likes": 50,
+          "comments": 5,
+          "date_posted": "2022-04-25"
+        }
+      },
+      {
+        "_index": "blogs",
+        "_id": "4",
+        "_score": 0.4352753,
+        "_source": {
+          "name": "A very old blog",
+          "views": 100,
+          "likes": 20,
+          "comments": 3,
+          "date_posted": "2000-04-25"
+        }
+      }
+    ]
+  }
+}
+```
+</details>
+
+### Example: Date fields
+
+The following query uses the Gaussian decay function to prioritize blogs published around 04/24/2002:
+
+```json
+GET blogs/_search
+{
+  "query": {
+    "function_score": {
+      "functions": [
+        {
+          "gauss": {
+            "date_posted": { 
+              "origin":  "2022-04-24",
+              "offset": "1d",
+              "scale":  "6d",
+              "decay": 0.25
+            }
+          }
+        }
+      ]
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+In the results, the first blog was published within one day of 04/24/2022, so it has the highest score of 1. The second blog was published on 04/17/2022, which is within `offset` + `scale` (`1d` + `6d`) and therefore has the score equal to `decay` (0.25). The third blog was published after 7 days of 04/22/2022, so it has a lower score. The last blog has a score of 0 because it was published years ago:
```

Comment:
Should this read 04/24 instead of 04/22? Or, should there be a code example using 04/22?